scene/gui/color_picker.cpp

```diff
@@ -398,25 +416,71 @@ void ColorPicker::_copy_hsv_to_color() {
 	}
 }
 
+void ColorPicker::_add_recent_preset_button(const bool selecte_from_preset_container) {
+	if (recent_preset_hbc->get_child_count() == 9) {
+		recent_preset_hbc->remove_child(recent_preset_hbc->get_child(8));
+	}
+	ColorPresetButton *btn_preset = memnew(ColorPresetButton(color));
+	int size = _get_preset_size();
+	btn_preset->set_toggle_mode(true);
+	btn_preset->set_preset_color(color);
+	btn_preset->set_custom_minimum_size(Size2(size, size));
```

Comment:
This looks like something that could be in the button's constructor (it's a custom class anyway).